odoo_module_migrate/migration.py

```diff
@@ -98,53 +99,70 @@ def __init__(
         # get migration scripts, depending to the migration list
         self._get_migration_scripts()
 
+    def _execute_shell(self, shell_command, **kwargs):
```

Comment:
I don’t really see the advantage in this patch to the method… IMHO, it ends up complicating a bit the different methods and the overall structure… I would leave it as is, passing the `pass` keyword argument each time…

Reply:
@ivantodorovich what is your point of view, regarding that comment ? 

thanks !

Reply:
It's to avoid having to pass the directory_path every time (and to avoid issues in case we forget about it)
For example, see the issue we had here: https://github.com/OCA/odoo-module-migrator/pull/64

As when running migrations, we mostly want to execute the shell in the directory of the repository (provided by the `--directory` argument)